reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,257 @@
+
+Windows PowerShell 5.1 and below use the `WS-Management` protocol for connection
+negotiation and data transport. If remoting has been
+enabled, installing PowerShell 7 will use the existing **WinRM** configuration.
+
+> [!NOTE]
+> PowerShell 7 will use the existing Windows PowerShell 5.1 endpoint for remoting
+> connections. To update PowerShell 7 to include it own endpoint, run the
+> `Install-PowerShellRemoting.ps1` located in **$PSHOME**. For information about connecting to
+> specific endpoints, see
+> [WS-Management Remoting in PowerShell Core](/powershell/scripting/learn/remoting/wsman-remoting-in-powershell-core?view=powershell-7)
+
+To use Windows PowerShell remoting, the remote computer must be configured for remote management.
+For more information, including instructions, see
+[About Remote Requirements](/powershell/module/microsoft.powershell.core/about/about_remote_requirements?view=powershell-7).
+
+For more information about working with remoting, see [About Remote](/powershell/module/microsoft.powershell.core/about/about_remote?view=powershell-7)
+
+### SSH-based remoting
+
+Starting with PowerShell Core 6.x to include cross-platform operating systems that can't use a
+Windows native components like **WinRM**, **SSH-based** remoting is now available to support
+cross-platform management.
+
+SSH remoting creates a PowerShell host process on the target computer as an SSH subsystem. For
+details and examples on setting up Windows or Linux for SSH-based remoting, see:
+[PowerShell remoting over SSH](/powershell/scripting/learn/remoting/ssh-remoting-in-powershell-core?view=powershell-7).
+
+> [!NOTE]
+> The PowerShell Gallery (PSGallery) contains a module and cmdlet that will automatically
+> configure **SSH-based** remoting. You can install the `Microsoft.PowerShell.RemotingTools` module
+> from
+> [PSGallery](https://www.powershellgallery.com/packages/Microsoft.PowerShell.RemotingTools/0.1.0) >
+> and execute the `Enable-SSH` cmdlet.
+
+The `New-PSSession`, `Enter-PSSession`, and `Invoke-Command` cmdlets now have a new parameter set to
+support this new remoting connection.
+
+```powershell
+[-HostName <string>]  [-UserName <string>]  [-KeyFilePath <string>]
+```
+
+To create a remote session, specify the target computer with the **HostName** parameter and provide
+the user name with **UserName**. When running the cmdlets interactively, you're prompted for a
+password.
+
+```powershell
+Enter-PSSession -HostName <Computer> -UserName <Username>
+```
+
+Alternatively, When using the **HostName** parameter, provide the username information followed by
+the **@** sign, followed by the computer name.
+
+```powershell
+Enter-PSSession -HostName <Username>@<Computer>
+```
+
+You may setup **SSH** key authentication using a private key file with the KeyFilePath parameter.
+For more information, see;
+[OpenSSH Key Management](https://docs.microsoft.com/windows-server/administration/openssh/openssh_keymanagement).
```

Comment:
It's not a blog, but it probably still needs a sign-off of some kind. Idk how @sdwheeler feels about call-to-action type stuff in docs, but it might be worthwhile to mention this is a living doc, and that we plan to add some stuff (and maybe that folks should file issues for what they still need?)

Also, might be a larger endeavor, wouldn't necessarily block the publishing of this on it, but some guidance on using the PSSA compat rules, as well as ApiPort for .NET-based module authors. Both of those are a little more niche (the latter moreso than the former), but I think folks looking to move from 5.1 would find it helpful. 

Also missing the cross-platform considerations for the hybrid/heterogenous admins, but I consider that a lower pri as well.

Reply:
I like the call to action. I think the PSSA, ApiPort, and cross-plat topics should follow on articles from this.